src/server/controllers/movie.js

```diff
@@ -1,26 +1,32 @@
-const jwt = require('jsonwebtoken');
 const { PrismaClient } = require('@prisma/client');
 const prisma = new PrismaClient();
 
-const jwtSecret = 'mysecret';
-
 const getAllMovies = async (req, res) => {
-    const movies = await prisma.movie.findMany();
+    const userId = parseInt(req.userId)
+    const movies = await prisma.movie.findMany({
+        where: {
+            userId: userId
+        }
+    });
 
     res.json({ data: movies });
 };
 
 const createMovie = async (req, res) => {
     const { title, description, runtimeMins } = req.body;
-
-    try {
-        const token = null;
-        // todo verify the token
-    } catch (e) {
-        return res.status(401).json({ error: 'Invalid token provided.' })
-    }
-
-    const createdMovie = null;
+    const userId = parseInt(req.userId)
```

Comment:
this id is being passed from the bearer token by my middlewear function where i verfiy the token. i suppose since its stored in the token as a string, it still stays as a string when i verify it. 
<img width="812" alt="Screenshot 2022-04-21 at 11 09 49" src="https://user-images.githubusercontent.com/91895565/164432806-7a357dc2-a9e0-4dbd-89be-66624af9c1aa.png">